recipe/meta.yaml

```diff
@@ -103,8 +103,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - khallock
-    - marylhaley
-    - david-ian-brown
-    - ocefpaf
-    - xylar
```

Comment:
I would keep @ocefpaf and @xylar on here for sure, I believe they are both still active in maintaining this package.

Reply:
Barely maintaining, sure, but we are doing what we can to keep the builds alive :smile:

Reply:
Sure, sorry @ocefpaf for missing that! I just didn't want to bother people who are not officially in the maintaining position. How about keeping you and the others, @khallock ?

Reply:
Usually we ask folks to either remove themselves or ask to be removed. The reason is b/c once you do remove them, they will lose access to the feedstock and can no longer merge PR. If they are inactive is not a big deal, specially if they silenced the notifications.

Reply:
Restored the original maintainers list